prime-router/src/main/kotlin/Report.kt

```diff
@@ -345,8 +358,7 @@ class Report {
 
             // Build sources
             val sources = inputs.map { ReportSource(it.id, "merge") }
-
-            return Report(schema, newTable, sources)
+            return Report(schema, newTable, sources, destination = head.destination, bodyFormat = head.bodyFormat)
         }
```

Comment:
Likewise, I'm making `merge` set merge set `destination` and `bodyFormat` so it matches the input reports.